src/test/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.test.tsx

```diff
@@ -215,15 +215,15 @@ describe('<AutomatedAnalysisCard />', () => {
   it('renders report generation error view correctly', async () => {
     jest.spyOn(defaultServices.api, 'graphql').mockReturnValueOnce(of(mockActiveRecordingsResponse));
     jest.spyOn(defaultServices.reports, 'reportJson').mockReturnValueOnce(of());
-    renderWithServiceContextAndReduxStore(<AutomatedAnalysisCard />, {
+    renderWithServiceContextAndReduxStore(<AutomatedAnalysisCard dashboardId={0} />, {
       preloadState: preloadedState,
     });
 
     expect(screen.getByText('Automated Analysis Error')).toBeInTheDocument(); // Error view
     expect(screen.getByText(FAILED_REPORT_MESSAGE)).toBeInTheDocument(); // Error message
     expect(screen.getByText('Cryostat was unable to generate an automated analysis report.')).toBeInTheDocument(); // Error details
     expect(screen.getByRole('button', { name: /retry loading report/i })).toBeInTheDocument(); // Retry button
-    expect(screen.queryByLabelText('automated-analysis-toolbar')).not.toBeInTheDocument(); // Toolbar
+    expect(screen.queryByLabelText('Automated Analysis Toolbar')).not.toBeInTheDocument(); // Toolbar
```

Comment:
Should use `testTranslate` to pull translated strings? Same for other tests.

Reply:
Not sure, the tests work fine without it, probably because the component gets rendered with i18n correctly before being tested? What should the right behaviour be here?

Reply:
I think because `Common.tsx` already intialize the i18n module by side-effect importing it. This mock i18n module is always set to english so it passed. I think we might need to more explicit by calling `testTranslate`?